packages/playwright/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-playwright
+_pkgname=playwright-python
```

Comment:
```suggestion
_pkgname=${pkgname#python-}
```

Reply:
I was using the entire "playwright-python" because it was the name of the GitHub project.